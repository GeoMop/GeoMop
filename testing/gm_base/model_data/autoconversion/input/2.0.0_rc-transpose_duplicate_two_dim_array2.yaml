flow123d_version: 2.0.0_rc
problem: !Coupling_Sequential
  description: Test
  mesh:
    mesh_file: mesh.msh
  flow_equation: !Flow_Darcy_MH
  
    # Test data
    input_fields:
      # Transpose input_fields to array with two items
      rid: [1, 2]
      # and convert anisotropy into a single item two dimensional array
      # and dupliacate anisotropy values
      anisotropy: [2.0, 3.0, 4.0]


    # Placeholder data
    nonlinear_solver:
      linear_solver: !Petsc
        r_tol: 1
        a_tol: 1
    balance: 
      balance_on: true
    output:
      output_stream: 
        file: ./test1.pvd
        format: !gmsh
      output_fields:
        - pressure_diff
