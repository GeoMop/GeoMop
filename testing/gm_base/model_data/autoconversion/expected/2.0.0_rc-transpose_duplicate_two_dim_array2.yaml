flow123d_version: 2.0.0_rc
problem: !Coupling_Sequential
  description: Test
  mesh:
    mesh_file: mesh.msh
  flow_equation: !Flow_Darcy_MH
  
    # Test data
    input_fields:
      # Transposed input_fields to array with two items
      - rid: 1
        # and converted anisotropy into a single item two dimensional array
        # and dupliacated anisotropy values
        # TODO: is this the expected behavior?
        anisotropy: !FieldConstant
          value: 
            - [2.0]
            - [3.0]
            - [4.0]
            
      - rid: 2
        anisotropy: !FieldConstant
          value: 
            - [2.0]
            - [3.0]
            - [4.0]            

    # Placeholder data
    nonlinear_solver:
      linear_solver: !Petsc
        r_tol: 1
        a_tol: 1
    balance: 
      balance_on: true
    output:
      output_stream: 
        file: ./test1.pvd
        format: !gmsh
      output_fields:
        - pressure_diff
